src/oic/utils/sdb.py

```diff
@@ -497,7 +497,7 @@ def _get_token_type_and_key(self, item, order=None):
         for key in order:
             try:
                 return self.token_factory[key].type_and_key(item)
-            except Exception:
+            except Exception:  # nosec
```

Comment:
We should really get to these at some point and fix them to be more specific. Could you please add a FIXME, so we do not forget?

Reply:
ok.